src/api/users.controller.js

```diff
@@ -0,0 +1,72 @@
+const { usersService } = require('../services/users.service.js');
+
+const getUsers = (req, res) => {
+  const users = usersService.getUsers();
+
+  res.status(200).json(users);
+};
+
+const getUserById = (req, res) => {
+  const user = usersService.getUserById(+req.params.id);
+
+  if (!user) {
+    return res.sendStatus(404);
+  }
+
+  res.json(user);
+};
+
+const createUser = (req, res) => {
+  const name = req.body.name;
+
+  if (!name) {
+    return res.status(400).json({ message: 'Name is required' });
+  }
+
+  const user = usersService.createUser(name);
+
+  res.status(201).json(user);
+};
+
+const deleteUserById = (req, res) => {
+  const user = usersService.deleteUserById(+req.params.id);
```

Comment:
Ensure that `usersService.deleteUserById` correctly returns a value indicating whether the deletion was successful. If it returns `undefined` or `null`, the current check will work, but if it returns a boolean, you might need to adjust the logic.